data/binding/gen.go

```diff
@@ -308,12 +308,12 @@ const listBindTemplate = `
 type {{ .Name }}List interface {
 	DataList
 
-	Append({{ .Type }}) error
+	Append(val {{ .Type }}) error
 	Get() ([]{{ .Type }}, error)
-	GetValue(int) ({{ .Type }}, error)
-	Prepend({{ .Type }}) error
-	Set([]{{ .Type }}) error
-	SetValue(int, {{ .Type }}) error
+	GetValue(index int) ({{ .Type }}, error)
+	Prepend(val {{ .Type }}) error
+	Set(v []{{ .Type }}) error
```

Comment:
If this change is for readability I'd suggest that the `v` should be `val` to match the other additions?

Reply:
Agree, initially I chose `v` for `Set(...)`, because I wanted to difference singular values from composite ones (slices), but maybe this is less clear? Or should we use `l` (**l**ist)?

Reply:
What is wrong with using `list`? or `values` something that make it clear there is a difference.

Reply:
Agree :) I will update.

Reply:
@andydotxyz Updated :)